### ISSUE TITLE
Reporting & Dashboard filtered by RCO country

### ISSUE THREAD

**Issue by karentom** [springernature/snics-management#316]
Can we add a drop-down for the dashboard functionality to allow the RCO to filter by country?

**Comment by harshadporwal** [comment]
@karentom I need more clarification for this requirment

**Comment by karentom** [comment]
@harshadporwal 
For example the dropdown should say 'Filter by Country' and then the pie charts will change according to the country selected. There should be an option for 'All' as well.

![image](https://user-images.githubusercontent.com/17595871/55557133-2b7b1000-56b7-11e9-8553-868081aa38d6.png)

**Comment by harshadporwal** [comment]
@karentom Added the country filter for Assessment staus chart. Please test it in UAT.

**Comment by karentom** [comment]
@harshadporwal The dropdown should impact all of the pie charts.

Also, Steph would like the ability to select multiple countries. For example, if the RCO wants to only see India and the United States, they can select both.

**Comment by karentom** [comment]
@harshadporwal if the multi-select for countries is too difficult, we can look at this at a later date. We can deploy the one country select dropdown to live for now.

**Comment by harshadporwal** [comment]
@karentom For depoyment once the admin role and read-only testing is done then only I can otherwise it will cause issue as the there fucntionality is also there is UAT.

**Comment by karentom** [comment]
@harshadporwal I will create a new issue for the multi-select functionality. The single select looks good and I will close this issue.
